mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -489,6 +611,28 @@ func flattenAPIKeys(keys []*apiKey) []map[string]interface{} {
 	return res
 }
 
+func flattenLimits(limits []admin.DataFederationLimit) []map[string]interface{} {
+	res := make([]map[string]interface{}, len(limits))
+
+	for i, limit := range limits {
```

Comment:
+100. Nice pattern for not including 0 values. 
We could use `limit.GetCurrentUsage()` but it would mean that 0 values will be introduced.

Reply:
exactly, wanted to avoid setting undefined values